Makefile

```diff
@@ -477,11 +483,23 @@ envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
 
-.PHONY: bundle
-bundle: manifests kustomize set-image ## Generate bundle manifests and metadata, then validate generated files.
+.PHONY: set-image-ubi
+set-image-ubi: kustomize copy-config ## Set the controller UBI image
+	cd $(CONFIG_MANAGER_DIR) && $(KUSTOMIZE) edit set image controller=$(IMG_UBI)
+
+.PHONY: sdk-generate
+sdk-generate:
 	operator-sdk generate kustomize manifests -q
-	$(KUSTOMIZE) build $(KUSTOMIZE_BUILD_DIR)/manifests | operator-sdk generate bundle $(BUNDLE_GEN_FLAGS)
-	operator-sdk bundle validate ./bundle
+
+.PHONY: bundle
+bundle: manifests kustomize sdk-generate set-image-ubi ## Generate bundle manifests and metadata, then validate generated files.
+	@rm -rf $(BUNDLE_DIR)
+	@rm -f bundle.Dockerfile
```

Comment:
nit: any chance we could move the `bundle.Dockerfile` under `build/bundle`? Even though we are ignoring it it could could still lead to some confusion since it persists in the repo root.

Reply:
Yeah it's stashed under `build` now in d1c2c8a.